### ISSUE TITLE
Remove minimum character-count limitation on comments with attached images

### ISSUE THREAD

**Issue by rEaL-jAsE** [devRant/devRant#504]
I'd like to have the minimum of 1 character limitation removed. Sometimes the best response to a rant or another comment is an image. And if the best response is an image, then no text content is necessary. Most of us use a dot to bypass this, which is very annoying. And it looks weird when there is a dot right above a meme/picture. So why not just remove that limitation if an image has been attached?

**Comment by Skayo** [comment]
Sounds like a good idea!

**Comment by Ewpratten** [comment]
or add comment types and have a selector in the style of the rant selector. like: [Image] [Everything else]

**Comment by Skayo** [comment]
@Ewpratten Sounds unnecessarily complicated

**Comment by rEaL-jAsE** [comment]
@Ewpratten I'm all-for types and categorization, but the problem is, my memory is really shit lately, and I often forget to properly categorize my own rants. I fail to see how I'll ever get into the habit of categorizing my comments. I might even become bothered by it, and stop commenting as much. Why do we need to categorize them [manually] anyway? It could be done automagically.

**Comment by Ewpratten** [comment]
True. And it would be nice if the API had a value to say if the comment contains text or not.

**Comment by rEaL-jAsE** [comment]
@Ewpratten that would b good. is the api public yet? like, do we have docs and stuff?

**Comment by 7twin** [comment]
@rEaL-jAsE not public (yet), but many endpoints are documented by skayo: https://devrant-docs.github.io everything else can be reversed by using mitmproxy.
